examples/flutter_gallery/lib/demo/fortnightly/fortnightly.dart

```diff
@@ -0,0 +1,236 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class FortnightlyDemo extends StatelessWidget {
+  static const String routeName = '/fortnightly';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Fortnightly Demo',
+      theme: _fortnightlyTheme,
+      home: Scaffold(
+        body: Stack(
+          children: <Widget>[
+            FruitPage(),
+            SafeArea(
+              child: ShortAppBar(
+                onBackPressed: () {
+                  Navigator.pop(context);
+                },
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class ShortAppBar extends StatelessWidget {
+  const ShortAppBar({ this.onBackPressed });
+
+  final VoidCallback onBackPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 96,
+      height: 50,
+      child: Material(
+        color: Theme.of(context).colorScheme.surface,
+        elevation: 4,
+        shape: const BeveledRectangleBorder(
+          borderRadius: BorderRadius.only(bottomRight: Radius.circular(22)),
+        ),
+        child: Row(
+          children: <Widget>[
+            IconButton(
+              icon: const Icon(Icons.arrow_back),
+              tooltip: 'Back',
+              onPressed: onBackPressed,
+            ),
+            const SizedBox(width: 12),
+            Image.asset(
+              'logos/fortnightly/fortnightly_logo.png',
+              package: 'flutter_gallery_assets',
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class FruitPage extends StatelessWidget {
+  final String paragraph1 = '''Have you ever held a quince? It\'s strange;
```

Comment:
Could be `static final String _paragraph1`

Reply:
Done!